onformsubmit-http/utils/submission.js

```diff
@@ -13,27 +14,37 @@ class Submission {
       this._submissionTime = message.submissionTime;
       this._email = message.email;
       this._editLink = message.editLink;
-      this._qaMap = new Map();
+
+      // deep copy and remove bulk QA data
+      this._json = JSON.parse(JSON.stringify(message));
+      delete this._json.questions;
+      delete this._json.responses;
```

Comment:
this ok?

Reply:
This is fine. Another way to do it would be to merge another object like`{ ...message ...{questions: undefined, responses: undefined}}`